### DISCUSSION TITLE
Can we specify the unique together constraint in models?

### DISCUSSION THREAD

**Discussion by harshalizode** [dymmond/mongoz#36] [Q&A] [answered]
Hello,
I would like to set the unique-together constraint in the class Meta as the Django Meta.
How can we achieve this using indexing or another mechanism?
Thank you.

**Comment by harshalizode** [comment]
We can set the unique-together constraint in meta by creating the index as:
```
indexes = [
    Index(keys=[("name", Order.DESCENDING), ("email", Order.DESCENDING)], unique=True),
]
```

**Reply by tarsil** [reply]
@harshalizode did you solve that problem?

**Reply by harshalizode** [reply, marked as answer]
Yes, Mongoz has the feature to set the unique-together constraint in the Class Meta, but not have the documentation of it :)

**Reply by tarsil** [reply]
Odd. Good to know. I will update the documentation for it as I missed it. Apologies and thank you for it.
